sklearn/impute/_base.py

```diff
@@ -482,11 +500,12 @@ class MissingIndicator(TransformerMixin, BaseEstimator):
     """
 
     def __init__(self, missing_values=np.nan, features="missing-only",
-                 sparse="auto", error_on_new=True):
+                 sparse="auto", error_on_new=True, precomputed=False):
```

Comment:
We don't need a precomputed flag. precomputed can be determined by X having a Boolean dtype and missing_values=True

Reply:
So what do you suggest, should I replace it with the mentioned conditions every place, or should I remove the parameter and define a data member precomputed,  as this second approach would make thing easy to read and maintain.

Thanks.